openapi.yaml

```diff
@@ -0,0 +1,675 @@
+        metadata:
+          $ref: '#/components/schemas/Object'
+        free:
+          type: boolean
+          default: true
+        bindable:
+          type: boolean
+        schemas:
+          $ref: '#/components/schemas/SchemasObject'
+
+    SchemasObject:
+      properties:
+        service_instance:
+          $ref: '#/components/schemas/ServiceInstanceSchemaObject'
+        service_binding:
+          $ref: '#/components/schemas/ServiceBindingSchemaObject'
+
+    ServiceInstanceSchemaObject:
+      properties:
+        create:
+          $ref: '#/components/schemas/SchemaParameters'
+        update:
+          $ref: '#/components/schemas/SchemaParameters'
+
+    ServiceBindingSchemaObject:
+      properties:
+        create:
+          $ref: '#/components/schemas/SchemaParameters'
+
+    SchemaParameters:
+      properties:
+        parameters: 
+          $ref: '#/components/schemas/JSONSchemaObject'
+
+    JSONSchemaObject:
+      type: object
+
+    ServiceInstanceProvisionRequest:
+      required:
+        - service_id
+        - plan_id
+        - organization_guid
+        - space_guid
+      properties:
+        service_id:
+          type: string
+        plan_id:
+          type: string
+        context:
+          $ref: '#/components/schemas/Object'
+        organization_guid:
+          type: string
+          deprecated: true
+        space_guid:
+          type: string
+          deprecated: true
+        parameters:
+          $ref: '#/components/schemas/Object'
+
+    ServiceInstanceProvision:
+      properties:
+        dashboard_url:
+          type: string
+        operation:
+          type: string # could be a link object to last operation
+
+    ServiceInstanceUpdateRequest:
+      required:
+        - service_id
+      properties:
+        context:
+          $ref: '#/components/schemas/Object'
+        service_id:
+          type: string
+        plan_id:
+          type: string
+        parameters:
+          $ref: '#/components/schemas/Object'
+        previous_values:
+          $ref: '#/components/schemas/ServiceInstancePreviousValues'
+
+    ServiceInstancePreviousValues:
+      properties:
+        service_id:
+          type: string
+          deprecated: true
+        plan_id:
+          type: string
+        organization_id:
+          type: string
+          deprecated: true
+        space_id:
+          type: string
+          deprecated: true
+
+    AsyncOperation:
+      required:
+        - operation
+      properties:
+        operation:
+          type: string
+
+    ServiceInstanceLastOperation:
+      required:
+        - state
+      properties:
+        state:
+          type: string
+          enum:
+            - in progress
+            - succeeded
+            - failed
+        description:
+          type: string
+
+    ServiceBindingRequest:
+      required:
+        - service_id
+        - plan_id
+      properties:
+        context:
+          $ref: '#/components/schemas/Object'
```

Comment:
Is there an easy way to link to the [Context Conventions](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#context-object) here? Maybe via a `description`?

Reply:
done